docs/.vitepress/vitepress/components/globals/overview.vue

```diff
@@ -25,6 +25,8 @@
           <el-card
             v-for="(item, index) in group.children"
             :key="index"
+            tabindex="0"
+            :data-link="item.link"
             shadow="hover"
             @click="toPage(item.link)"
           >
```

Comment:
```suggestion
          <el-card
            v-for="(item, index) in group.children"
            :key="index"
            tabindex="0"
            shadow="hover"
            @click="toPage(item.link)"
            @keydown.enter="toPage(item.link)"
          >
```
You don't need to write complicated.

Reply:
Makes sense. 👍